src/libs/runtime-utils.js

```diff
@@ -381,8 +381,8 @@ export const RadioBlock = ({ labelText, children, name, checked, onChange, style
 }
 
 export const getIsAppBusy = app => app?.status !== 'RUNNING' && _.includes('ING', app?.status)
-export const getIsRuntimeBusy = runtime => {
-  const { Creating: creating, Updating: updating, LeoReconfiguring: reconfiguring } = _.countBy(getConvertedRuntimeStatus, [runtime])
-  return creating || updating || reconfiguring
+export const getIsRuntimeBusy = (runtime, isErrorBusy = false) => {
```

Comment:
With `isErrorBusy` set to true, this function's name is no longer descriptive. Especially since it looks like this is only used in one place, I think it would be clearer to use something like `isRuntimeBusy || isRuntimeError` or `isRuntimeInState(["creating", "updating", ...])` instead of adding this flag.

https://alexkondov.com/should-you-pass-boolean-to-functions/

Reply:
Thanks. I find the use of this function in ComputeModal.js much easier to understand after d10f106f9a95f0fb78894d8c8c3aa08af9540d8c.